src/main/scala/scalaz/DistributedModule.scala

```diff
@@ -43,10 +52,17 @@ trait DistributedModule {
   }
 
   trait Metadata[A] {
-    def set[B: Type](where: Lens[A, B], b: B)(implicit A: Type[A]): F[Unit]
-    def get[B: Type](where: Lens[A, B])(implicit A: Type[A]): F[B]
-    def increment(where: Lens[A, Int])(implicit A: Type[A]): F[Unit]
+    def set[B: Type](where: Path[A, B], b: B)(implicit A: Type[A]): F[Unit]
+
+    def modify[B: Type](where: Path[A, B], f: B => B)(implicit A: Type[A]): F[Unit] =
```

Comment:
I think `modify` can be `final` as well.

Reply:
done

Reply:
done